web/apps/labelstudio/src/themes/default/colors.scss

```diff
@@ -93,12 +81,10 @@
   --primary-bright-secondary: var(--grape_0);
   --primary-bright-tertiary: var(--grape_100);
   --primary-text-on-background-dark: var(--grape_800);
-
   --accent_color: var(--grape_500);
   --colors-surface-border: var(--sand_300);
   --colors-surface-icon-on-base: var(--sand_700);
   --colors-surface-text: var(--sand_700);
-
-  --primary-button-waiting-animation-bg: repeating-linear-gradient(-63.43deg, hsla(0, 0%, 100%, 0.2) 1px, transparent 2px, transparent 6px, hsla(0, 0%, 100%, 0.2) 7px, hsla(0, 0%, 100%, 0.2) 12px);
-  --button-waiting-animation-bg: repeating-linear-gradient(-63.43deg,hsla(60,20%,99%,.2) 1px,#efefef 2px,#efefef 6px,hsla(60,20%,99%,.2) 7px,hsla(60,20%,99%,.2) 12px);
-}
\ No newline at end of file
+  --primary-button-waiting-animation-bg: repeating-linear-gradient(-63.43deg, hsl(0deg 0% 100% / 20%) 1px, transparent 2px, transparent 6px, hsl(0deg 0% 100% / 20%) 7px, hsl(0deg 0% 100% / 20%) 12px);
+  --button-waiting-animation-bg: repeating-linear-gradient(-63.43deg,hsl(60deg 20% 99% / 20%) 1px,#efefef 2px,#efefef 6px,hsl(60deg 20% 99% / 20%) 7px,hsl(60deg 20% 99% / 20%) 12px);
```

Comment:
interesting that we got hsl colours here instead of rgb, hopefully our work with design tokens will help standardize our colours